src/include/processor/processor_task.h

```diff
@@ -15,7 +15,6 @@ class ProcessorTask : public common::Task {
     void run() override;
     void finalizeIfNecessary() override;
 
-private:
     static std::unique_ptr<ResultSet> populateResultSet(Sink* op,
         storage::MemoryManager* memoryManager);
```

Comment:
can you add back the `private` below this function so that the members are still private?

Reply:
There is a `private` below.